website/react-magma-docs/src/pages/api/chart.mdx

```diff
@@ -33,14 +39,12 @@ We currently support the following chart types:
 
 ## Area Chart
 
-Area charts share similarities with line charts, distinguished by the filled areas below the lines, adorned with colors or patterns. Stacked charts, on the other hand, prove valuable for comparing proportional contributions within a category by plotting the relative value each data series contributes to the total. 
+Area charts share similarities with line charts, distinguished by the filled areas below the lines, adorned with colors or patterns. Stacked charts, on the other hand, prove valuable for comparing proportional contributions within a category by plotting the relative value each data series contributes to the total.
 
 ```typescript
 import React from 'react';
 import { Card } from 'react-magma-dom';
 import { CarbonChart, CarbonChartType } from '@react-magma/charts';
-// IMPORTANT: Uncomment the line below for the correct styles
-// import '@react-magma/charts/dist/charts.css';
```

Comment:
We do want to keep these lines in all the examples -- they are for adopters and will not break the build